.vscode/settings.json

```diff
@@ -1,7 +1,14 @@
 {
   "tailwindCSS.experimental.configFile": {
-    "apps/storybook-react/tailwind.config.js": "packages/design-system-react/src/**",
-    "apps/storybook-react-native/tailwind-intellisense.config.js": "packages/design-system-react-native/src/**"
+    "apps/storybook-react/tailwind.config.js": [
+      "apps/storybook-react/**",
+      "packages/design-system-react/src/**",
+      "packages/design-tokens/stories/**"
+    ],
+    "apps/storybook-react-native/tailwind-intellisense.config.js": [
+      "packages/design-system-react-native/src/**",
+      "apps/storybook-react-native/stories/**"
```

Comment:
Tailwind intellisense and prettier work as expected in `storybook-react-native`


https://github.com/user-attachments/assets/cb63992e-a61e-4956-a67b-593f702a4d49